docs/en/deployment.md

```diff
@@ -9,199 +9,151 @@ Before you launch the solution, review the architecture, supported regions, and
 
-### Get QuickSight Username
+* Sign in to the AWS Management Console, navigate to [QuickSight](https://quicksight.aws.amazon.com/).
 
-* (Optional) Navigate to [QuickSight](https://quicksight.aws.amazon.com/), fill your email in below screen
+* Choose **Enterprise**, click continue.
 
-!!! notice
-    This step is only necessary if you login your AWS console with different IAM user(not the user sign up with QuickSight).
+* In the **Create your QuickSight account** page, fill the necessary information:
 
 <center>
-![QuickSight email](./images/quicksight-no-admin-email.png)
-
 
-Figure 3: QuickSight email
+|      Field Name      |   Value |  Comment |
+|:--------------------|:-------------------| :-------------------|
+| Authentication method | Choose **Use IAM federated identities & QuickSight-managed users** | The default value|
+| Select a region | Choose **US East(N.Virginia)**  | The default value |
+| QuickSight account name | Any string value | Unique QuickSight account name|
+| Notification email address | your email | |
+| IAM Role| Choose **Use an existing role**, select the role: `qradd-quicksight-service-role`| |
 
 </center>
 
 
-* Go to [QuickSight Admin](https://us-east-1.quicksight.aws.amazon.com/sn/admin), record your **QuickSight Username**(not QuickSight account name) for your email.
+!!! Caution 
 
-<center>
-![QuickSight username](./images/quicksight_username.png)
+    If Quicksight service already used default role, you need to change it to the role created above in page [QuickSight Security & permissions](https://us-east-1.quicksight.aws.amazon.com/sn/admin#aws).
+    This action may influence other data under this QuickSight Account. Please check in advance.
 
+### Get QuickSight Username
 
-Figure 2: QuickSight username
-
-</center>
-
-## Deployment
+* (Optional) Navigate to [QuickSight](https://quicksight.aws.amazon.com/), fill your email if QuickSight user is not created for
+ your current AWS account.
 
-Use the following steps to deploy this solution on AWS.
+* Go to [QuickSight Admin](https://us-east-1.quicksight.aws.amazon.com/sn/admin), record your **QuickSight Username**(not QuickSight account name) for your email.
 
-### Launch the AWS CloudFormation template into your AWS account to deploy the solution
+## Step 2: Launch the AWS CloudFormation template into your AWS account to deploy the solution
 
-This automated AWS CloudFormation template deploys the solution in the AWS Cloud.
+1. Login AWS console, click [Launch Stack][template-url] to deploy solution via the AWS CloudFormation template.
+ 
+2. The template launches in the US West(Oregon) by default. To launch this solution in a different AWS Region(for example `us-east-1`), use the Region selector in the console navigation bar.
 
-* [Launch solution in AWS Standard Regions][template-url] to deploy solution via the AWS CloudFormation template.
+3. Please review and confirm the Amazon S3 URL in the **Create stack** page, then click **Next**.
```

Comment:
it's really confused the end user. how can I review the url provided by this documentation!

Reply:
added the template link